django_mfa/views.py

```diff
@@ -19,12 +21,12 @@ def configure_mfa(request):
     qr_code = None
     base_32_secret = None
     if request.method == "POST":
-        try:
-            base_32_secret = base64.b32encode(bytes(request.user.email, 'utf-8'))
-        except:
-            base_32_secret = base64.b32encode(request.user.email)
-        #IOS Google Authenticator doesn't like the ='s at the end of a secret code
-        totp_obj = totp.TOTP(base_32_secret.decode("utf-8").replace("=",""))
+        base_32_secret = base64.b32encode(
```

Comment:
Alternatively we can use:
```
base64.b32encode(os.urandom(10))
```

Which is a bit easier to read then `getrandbits` and also is potentially suitable for cryptographic use (although there is discussion about that).